text/0000-macro-hygiene-optout.md

```diff
@@ -0,0 +1,221 @@
+- Feature Name: macro_hygiene_optout
+- Start Date: 2018-07-05
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This feature introduces the ability to "opt-out" of the usual macro hygiene rules within definitions of [declarative macros][decl-macro], for designated identifiers or occurrences of identifiers. In other words, the feature will enable one to annotate occurrences of identifiers with macro call-site hygiene rather than the default definition-site hygiene.
```

Comment:
Could be good to mention here that "declarative macros" does not refer to `macro_rules!` (it is apparent if you click the link, but in the interest of not having to do so...)

Reply:
Fair point. I originally had this, but somehow removed it.